services/src/unit-test/java/uk/gov/moj/sdt/services/BulkFeedbackServiceTest.java

```diff
@@ -248,4 +226,30 @@ private void addValidIndividualRequest(final IBulkSubmission bulkSubmission, fin
         bulkSubmission.addIndividualRequest(individualRequest);
     }
 
+    @Test
+    void testSetBulkSubmissionDao() {
+
+        IBulkFeedbackService bulkFeedbackService = new BulkFeedbackService(mockBulkSubmissionDao, mockGlobalParameterCache);
+        IBulkSubmissionDao bulkSubmissionDaoMock = mock(BulkSubmissionDao.class);
+        ((BulkFeedbackService) bulkFeedbackService).setBulkSubmissionDao(bulkSubmissionDaoMock);
+
+        Object result = this.getAccessibleField(BulkFeedbackService.class, "bulkSubmissionDao",
+                                                IBulkSubmissionDao.class, bulkFeedbackService);
+
+        assertEquals(bulkSubmissionDaoMock, result, DAO_SHOULD_BE_SET_TO_OBJECT);
+    }
```

Comment:
The bulkFeedbackService class field can be used instead of creating a new instance in the test and the cast when calling the setBulkSubmissionDao() seems unnecessary.  This allows the test to be simplified to:

```
IBulkSubmissionDao bulkSubmissionDaoMock = mock(BulkSubmissionDao.class);
bulkFeedbackService.setBulkSubmissionDao(bulkSubmissionDaoMock);

IBulkSubmissionDao result = (IBulkSubmissionDao) getAccessibleField(BulkFeedbackService.class,
                                                                    "bulkSubmissionDao",
                                                                    IBulkSubmissionDao.class,
                                                                    bulkFeedbackService);

assertEquals(bulkSubmissionDaoMock, result, DAO_SHOULD_BE_SET_TO_OBJECT);
```

Reply:
done